internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/producer/ApacheKafkaProducerAdapter.java

```diff
@@ -151,7 +150,10 @@ public void close(int closeTimeOutMs, boolean doFlush) {
     }
     if (doFlush) {
       // Flush out all the messages in the producer buffer
-      producer.flush(closeTimeOutMs, TimeUnit.MILLISECONDS);
+      producer.flush();
```

Comment:
IDK. We could. I am tempted to say we could leave it like this (no time out), and have a `LiKafkaProducerAdapter` which is essentially the same as this one, but just overrides this function to plug in the time out params... WDYT?

Reply:
At the moment, we're using Kafka producer only in VPJ and Samza producer. So it might not be worth adding another adapter as it means we will need one more config to swap them.

Reply:
What do you mean? Aren't we using the producer everywhere we use the `VeniceWriter`? AFAIK, we use the VW in the server, DVC, controller, online producer...

Reply:
The close refactoring in #1014 solves this.